### ISSUE TITLE
Latest Release Tag?

### ISSUE THREAD

**Issue by casperwilkes** [laravelista/comments#178]
Hello. I noticed some API stuff in a previous commit that i'd like to pull into a project (https://github.com/laravelista/comments/pull/176) i'm working on. 
However, I've notice that the last release tag is [4.5.0](https://github.com/laravelista/comments/releases/tag/4.5.0) and that happened on 2021/03/15.
There are 12 commits merged into master since then, the last one being on 2021/10/30. 
It looks like there's been a bunch of significant changes and i'd like to use them. 
Thanks

**Comment by mabasic** [comment]
Yes, sorry about that. I planned to make a release, but forgot due to work stuff. I will try to make a release in few days.

If you need it asap, use the master branch since that will be the next release.
